test/auth_tokens_SUITE.erl

```diff
@@ -129,7 +129,7 @@ validation_test(_, ExampleToken) ->
     %% given
     Serialized = ?TESTED:serialize(ExampleToken),
     %% when
-    Result = ?TESTED:authenticate(Serialized),
+    Result = ?TESTED:authenticate(domain(), Serialized),
```

Comment:
Could we use `host_type()` instead of `domain()` for functions that actually expect host types? Ideally we could put a different value there like test type to make a better distinction between host types and domains and to cover the case when the two have different values.